### PR TITLE
Update wxGTK docs

### PR DESCRIPTION
**Pull request by vadz**
See [this ticket](https://trac.wxwidgets.org/ticket/18994) for motivation.

### CODE REVIEW

**Comment by oneeyeman1** [comment]
@vadz ,
I think we can add the Open Indiana to the list of supported GTK platform.

File: docs/gtk/install.md [hunk 1]
```diff
@@ -119,108 +76,93 @@ Now create your super-application myfoo.cpp and compile anywhere with
...
 will see linking errors. If this happens then you can work around the
-problem by setting LDFLAGS=-Wl,--export-all-symbols. Please also let us know
+problem by setting `LDFLAGS=-Wl,--export-all-symbols`. Please also let us know
 about it on the wx-dev mailing list.
 
-Create your configuration              {#gtk_config}
----------------------------
 
-Usage:
+Configure options                        {#gtk_options}
```
Comment (imciner2) [inline docs/gtk/install.md:96]: The configure option for disabling the EGL backend and using the X11 backend instead for the OpenGL canvas should be added to this list.

**Review by imciner2 — CHANGES_REQUESTED** [review]

**Review by paulcor — APPROVED** [review]

**Comment by PBfordev** [comment]
As I wrote elsewhere, I know nothing about Linux so my question is probably stupid and this is not usually done.

In addition to mentioning dependency on gtk development package, would it not be useful to list the major 3rd party dependencies that may not generally be installed?
E.g., `wxMediaCtrl` depends on GStreamer, version at least X.Y, `wxWebView` depends on webkitgtk version at least X.Y, `wxGLCanvas` ...

I understand this can be learnt by carefully inspecting config.log but still...

File: docs/gtk/install.md (hunk 1 above)
Comment (vadz) [reply docs/gtk/install.md:96]: Thanks, good idea, added now.

**Comment by vadz** [comment]
Thanks @PBfordev, I've added a couple of big libraries, as you've suggested. This is certainly incomplete, but OTOH I don't know if specifying all the dependencies here is worthwhile as it will almost certainly get out of date pretty quickly -- there are just too many things to keep track of.

**Comment by PBfordev** [comment]
I think that listing dependencies for `wxMediaCtrl `and ` wxWebView` could be enough (I guess it would be more complicated with OpenGL). 

I thought that perhaps all other 3rd party dependencies (zip, regex? expat, image formats...) fall back to the built-in versions when their system versions are not found.

**Pull request merged**